src/main/java/hudson/plugins/analysis/graph/GraphConfiguration.java

```diff
@@ -433,6 +493,8 @@ private void reset() {
         dayCount = DEFAULT_DAY_COUNT;
         graphType = DEFAULT_GRAPH;
         useBuildDate = DEFAULT_USE_BUILD_DATE;
+        parameterName = null;
```

Comment:
When no parameter and value is set and it is serialized to defaults file it will take form: 500!200!50!30!PRIORITY!0!!! . 
Now if you feed a string like this to StringUtils, i.e : 
String[] values = StringUtils.split(value, SEPARATOR);
It will have only 6 values. So there is no way to distinguish between empty string and null in defaults file. 
So i am not sure what you want me to do here ;)